src/main.rs

```diff
@@ -1,16 +1,29 @@
+use riv::cli::cli;
 use riv::program::Program;
+use std::convert::TryInto;
 
 fn main() -> Result<(), String> {
+    let args = cli()?;
     let ttf_context = sdl2::ttf::init().map_err(|e| e.to_string())?;
     let sdl_context = sdl2::init()?;
     let video = sdl_context.video()?;
-    let window = video
-        .window(
-            "riv",
-            video.display_bounds(0).unwrap().width(),
-            video.display_bounds(0).unwrap().height(),
-        )
-        .position_centered()
+
+    const INITIAL_TITLE: &str = "riv";
```

Comment:
Please put constants at the top of the file, outside of the scope of functions. I believe this is the canonical way.